docs/dev/ASCRead.md

```diff
@@ -0,0 +1,213 @@
+---
+layout: docs
+title: ASCRead
+category: h2drivers
+is_function: true
+description: ASC &rarr; Table
+prev_section: h2drivers
+next_section: CSVRead
+permalink: /docs/dev/ASCRead/
+---
+
+### Signatures
+
+{% highlight mysql %}
+ASCRead(VARCHAR path);
+ASCRead(VARCHAR path, BOOLEAN deleteTable);
+
+ASCRead(VARCHAR path, VARCHAR myTable);
+ASCRead(VARCHAR path, VARCHAR myTable, BOOLEAN deleteTable);
+
+ASCRead(VARCHAR path, INTEGER type);
+ASCRead(VARCHAR path, INTEGER type, BOOLEAN deleteTable);
+
+ASCRead(VARCHAR path, VARCHAR myTable, INTEGER type);
+ASCRead(VARCHAR path, VARCHAR myTable, INTEGER type, BOOLEAN deleteTable);
+
+ASCRead(VARCHAR path, VARCHAR myTable, GEOMETRY geomFilter, 
+        INTEGER downScaleInt, BOOLEAN asPolygons);
+ASCRead(VARCHAR path, VARCHAR myTable, GEOMETRY geomFilter, 
+        INTEGER downScaleInt, BOOLEAN asPolygons, BOOLEAN deleteTable);
+{% endhighlight %}
+
+### Description
+
+Import Esri ASCII Raster file as `POINT` or `POLYGON` geometries.
+
+Pixels are converted into `PointZ` (or `PolygonZ`) geometry with Z as the pixel value (stored as `DOUBLE`).
+
+Where:
+
+- `path` : the adress of the `.asc` file. This file may be zipped in a `.gz` file *(in this case, the `ASCRead` driver will unzip on the fly the `.gz` file)*, 
+- `myTable` : the name given to the resulting table,
+- `type` : indicates whether the `z` data type will be casted to INTEGER (`1`) or left as DOUBLE (`2` - default value),
+- `geomFilter` : extract only pixels that intersects the provided geometry envelope (`null` to disable filter),
```

Comment:
`null` or empty argument to disable filter

Reply:
Done